website/content/docs/secrets/aws.mdx

```diff
@@ -86,6 +97,26 @@ management tool.
     your AWS root account credentials. Instead generate a dedicated user or
     role.
 
+1.  Alternatively, configure the audience claim value and the role ARN to assume for plugin workload identity federation:
+
+    ```text
+    $ vault write aws/config/root \
+        identity_token_audience="<TOKEN AUDIENCE>" \
+        role_arn="<AWS ROLE ARN>"
+    ```
+
+    Vault's identity token provider will internally sign the plugin identity token JWT.
+    This token will be exchanged to establish a trust relationship with AWS and to obtain
```

Comment:
Suggest a rephrase here similar to what I suggested above. This reads like the token exchange is what establishes the trust relationship between Vault and AWS when it's actually a pre-configuration step (as noted right below this).

Reply:
Good catch! Updated to highlight that this is 'given a trust relationship...'